src/main/app/claims/models/claimStatus.ts

```diff
@@ -42,5 +42,6 @@ export enum ClaimStatus {
   CLAIMANT_REQUESTS_CCJ_AFTER_DEFENDANT_REJECTS_SETTLEMENT,
   ORDER_DRAWN,
   REVIEW_ORDER_REQUESTED,
-  INTENTION_TO_PROCEED_DEADLINE_PASSED
+  INTENTION_TO_PROCEED_DEADLINE_PASSED,
+  PAPER_RESPONSE
```

Comment:
Please rename to something like DEFENDANT_PAPER_RESPONSE as the content is related only to the defendant submitting something by paper.